command/commands/team.py

```diff
@@ -476,3 +486,67 @@ def delete_helper(self, team_name, user_id) -> ResponseTuple:
             logging.error("team delete unsuccessful")
             return f"Team delete was unsuccessful with " \
                    f"the following error: {e.data}", 200
+
+    def refresh_helper(self, user_id) -> ResponseTuple:
+        """
+        Ensure that the local team database is the same as GitHub's.
+
+        In the event that our local team database is outdated compared to
+        the teams on GitHub, this command can be called to fix things.
+        :return: return error message if user has insufficient permission level
+                 otherwise returns success messages with # of teams changed
+        """
+        num_changed = 0
+        num_added = 0
+        num_deleted = 0
+        modified = []
+        try:
+            command_user = self.facade.retrieve(User, user_id)
+            if not check_permissions(command_user, None):
+                return self.permission_error, 200
+            local_teams: List[Team] = self.facade.query(Team)
+            remote_teams: List[GithubTeam.Team] = self.gh.org_get_teams()
+            local_ids = dict((team.github_team_id, team)
+                             for team in local_teams)
+            remote_ids = dict((str(team.id),
```

Comment:
I hate to be the picky one but I don't like the idea of calling a dictionary by something expected to be its key ._.

Reply:
What do you mean by that?

Reply:
I think I understand. What should I call it instead?

Reply:
`local_teams_dict` and `remote_teams_dict` maybe?

Reply:
ohhh I get it now